Kernel/Net/Routing.cpp

```diff
@@ -102,7 +102,7 @@ void ARPTableBlocker::will_unblock_immediately_without_blocking(UnblockImmediate
     SpinlockLocker lock(m_lock);
     if (!m_did_unblock) {
         m_did_unblock = true;
-        m_mac_address = move(addr);
+        m_mac_address = static_cast<Optional<MACAddress>>(addr);
```

Comment:
Side note, the surrounding code looks very :yakthonk:, given that we seem to take a reference to an entry from a locked table here? Maybe we should already make this an `Optional<Whatever>` within the locked area.